### DISCUSSION TITLE
Go to the track it playing...

### DISCUSSION THREAD

**Discussion by Emilot** [jcorporation/myMPD#467] [Ideas]
JC when i am in the Queue card 

1) I want my browser to go to the track that is currently playing and not in the bottom or up in the page.
2) I find it very user-friendly to emphasize at the current playing song at the queue card. Maybe a green bottom line under the current playing track? The bold it isn't very well made and the eye cannot focus at the first glance. 

PS. It will be more accurate to rename "song" or the plural "songs", to "track" or "tracks".

Thank you!

**Comment by jcorporation** [comment]
1) There is a goto playing track button

2) changing text color could be an other option

3) track is used in mpd as synonym for tracknumber

**Reply by Emilot** [reply]
JC the Goto Playing track button exists only if i have a second page in Queue

**Reply by jcorporation** [reply]
Should it be also there if playing song is not in viewport?

**Reply by Emilot** [reply]
If i have a queue page with 80 tracks it should be easier to have a button or the page go itself automatically to the current playing track and not on the top/bottom page :)

**Comment by Emilot** [comment]
As for 2. option it would be "fancy" to have the ongoing "playing green bar" from Playback card, also to the current playing song in Queue card and different color as you suggest!!

just a thought...

**Reply by Emilot** [reply]
![mympd_green](https://user-images.githubusercontent.com/18086896/114024576-e2995b80-987c-11eb-84a9-98458fa03060.jpg)

for example...

**Reply by jcorporation** [reply]
Your mock-up locks nice.
